quantestpy/circuit.py

```diff
@@ -6,41 +6,17 @@
 
 import numpy as np
 
-from quantestpy import TestCircuit, operator
-from quantestpy.converter import (_cvt_openqasm_to_test_circuit,
-                                  _cvt_qiskit_to_test_circuit,
-                                  _is_instance_of_qiskit_quantumcircuit)
+from quantestpy import QuantestPyCircuit, operator
+from quantestpy.converter.all import cvt_all_circuit_to_quantestpy_circuit
 from quantestpy.exceptions import QuantestPyAssertionError, QuantestPyError
+from quantestpy.simulator.state_vector_circuit import \
+    cvt_quantestpy_circuit_to_state_vector_circuit
 
 ut_test_case = unittest.TestCase()
 
 
-def _get_test_circuit_from_input_circuit(circuit: Union[TestCircuit, str]) \
```

Comment:
This function is moved in `quantestpy/converter/all.py` because this will be used by other assert methods, too.